### PR TITLE
KAFKA-10770: Remove duplicate defination of Metrics#getTags

### PR DESCRIPTION
**Pull request by ArunParthiban-ST**
*Removal of duplicate code.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: clients/src/main/java/org/apache/kafka/common/metrics/internals/MetricsUtils.java
Comment (chia7712) [inline clients/src/main/java/org/apache/kafka/common/metrics/internals/MetricsUtils.java]: Could you set the initial capability?

File: clients/src/test/java/org/apache/kafka/common/metrics/internals/MetricsUtilsTest.java
Comment (chia7712) [inline clients/src/test/java/org/apache/kafka/common/metrics/internals/MetricsUtilsTest.java]: Could you use ```assertThrows```?

**Review by chia7712 — COMMENTED** [review]
@arunmahadevan Thanks for your patch. some minor comments are left. Please take a look :)

File: clients/src/test/java/org/apache/kafka/common/metrics/internals/MetricsUtilsTest.java
Comment (ArunParthiban-ST) [reply clients/src/test/java/org/apache/kafka/common/metrics/internals/MetricsUtilsTest.java]: Done

File: clients/src/main/java/org/apache/kafka/common/metrics/internals/MetricsUtils.java
Comment (ArunParthiban-ST) [reply clients/src/main/java/org/apache/kafka/common/metrics/internals/MetricsUtils.java]: Sure!

**Comment by ArunParthiban-ST** [comment]
@chia7712   Updated PR. Please take a look.

**Review by chia7712 — APPROVED** [review]
@ArunParthiban-ST Thanks for your patch. LGTM

I will merge it after this weekend (to wait for more reviews)

**Pull request merged**
